app/models/review.rb

```diff
@@ -0,0 +1,10 @@
+class Review < ActiveRecord::Base
+  belongs_to :spot
+  belongs_to :user
+
+  validates :rating, presence: true
+  validates :rating, numericality: { greater_than: 0, less_than: 6 }
+  validates :body, presence: true
+  validates :spot_id, presence: true
+  validates :user_id, presence: true
```

Comment:
These should be:

```
validates :spot, presence: true
validates :user, presence: true
```

That way, it'll check that there is a valid entry for this object in the db